docs/reference/mobile/android/cloud-deploy.md

```diff
@@ -2,56 +2,40 @@
 
 # Connect to a cloud deployment
 
-Before reading this document, make sure you are familiar with:
+<%(Callout message="
+Before reading this document, make sure you have read:
 
 * [Setting up Android Support for the GDK]({{urlRoot}}/reference/mobile/android/setup)
-* [Ways to test your Android client]({{urlRoot}}/reference/mobile/android/run-client)
 * [Development authentication flow](https://docs.improbable.io/reference/latest/shared/auth/development-authentication)
-* [Creating workers with WorkerConnector](https://docs.improbable.io/unity/alpha/reference/workflows/monobehaviour/creating-workers)
+")%>
 
-## Prepare your project to connect to a cloud deployment {#prepare}
+## Start a cloud deployment
```

Comment:
I'd be concerned that given the latter two headers are "options", it could feel like this is an option when its actually a pre-req for the other two